### ISSUE TITLE
figure out 'live' logging to dx logs

### ISSUE THREAD

**Issue by jethror1** [eastgenomics/eggd_conductor#12]
since everything is run with python, none of the prints go to dx_stdout until it finishes running so the app has nothing until it completes or fails, where all the stdout of python appears in one go. Would be better to have this actually keep writing to dx_stdout as it is running.

**Comment by jethror1** [comment]
can be done by writing to `dx_stdout` or `dx_stderr` instead of prints:
```
import logging as log

log.basicConfig(filename='dx_stdout', level=logging.DEBUG, format='')

log.info('logging a thing')
```

**Comment by jethror1** [comment]
Fixed in #24
